### DISCUSSION TITLE
Opening new tabs in the current kitten

### DISCUSSION THREAD

**Discussion by Lun4m** [kovidgoyal/kitty#6434] [Q&A] [answered]
I used to be able to open a new tab in the current kitten with
```
map ctrl+shift+t launch --cwd=oldest --type=tab    # open tab in kitten
map f1 launch --type=tab                           # If in a kitten, open tab locally
```
but now when I use `ctrl+shift+t` I get
```terminal
Failed to launch child: kitten
With error: No such file or directory
Press Enter or Esc to exit
```
I don't know if this is because I upgraded Ubuntu from 22.10 to 23.04, or something changed in kitty or the server I'm connecting to.
Does anyone have any pointers on how to track down where the problem is and how to fix it? Maybe related issues #5672 #6015.

**Comment by kovidgoyal** [comment]
Use an official binary of kitty from https://sw.kovidgoyal.net/kitty/binary/  ubuntua packages are very outdated/broken. I can tell you that this works fine with up-to-date kitty.

**Reply by Lun4m** [reply]
Sorry, I should've mentioned I'm using the official binary (just updated this morning to the latest version, but the problem was also happening before).

**Reply by kovidgoyal** [reply]
kitty --config=NONE -o 'map ctrl+shift+t launch --cwd=oldest --type=tab'

works fine for me.

**Reply by Lun4m** [reply]
I still get `Failed to launch child: kitten` when I try to open a new tab after connecting to the server with `kitty +kitten ssh server` (in the kitty instance started by the command you provided).

**Reply by kovidgoyal** [reply, marked as answer]
Make sure the kitten binary is in your PATH and you should be fine. And this needs to be in the PATH of whatever environment you are launching kitty from, not the environment inside your shell.

**Reply by kovidgoyal** [reply]
And this will remove the requirement of having kitten in PATH https://github.com/kovidgoyal/kitty/commit/d2bdfbee9dd0a29ad9e22de2ef147c7a2d913e24

**Reply by Lun4m** [reply]
That was it! Somehow `kitten` was not in my `.local/bin`.
Thanks for the help and for such a great terminal!
